### PR TITLE
Grafana alerts provisioning

### PR DESCRIPTION
**Pull request by Nutomic**
## Summary of changes
- Create alert if token balance is less than 1 Eth. Currently it doesnt account for tokens with different number of decimals
- Alert and Slack notification contact point are provisioned automatically from filesystem
- Daily alerts are not possible (see https://github.com/webb-tools/relayer/issues/389#issuecomment-1521783577)

### Reference issue to close (if applicable)
- Closes https://github.com/webb-tools/relayer/issues/389

-----
### Code Checklist 

- [ ] Tested
- [ ] Documented

### CODE REVIEW

File: tests/test/evm/vanchorPrivateTransaction.test.ts [hunk 1]
```diff
@@ -240,7 +240,7 @@ describe('Vanchor Private Tx relaying with mocked governor', function () {
     await webbRelayer.waitUntilReady();
   });
 
-  it('should relay private transaction', async () => {
+  it.only('should relay private transaction', async () => {
```
Comment (shekohex) [inline tests/test/evm/vanchorPrivateTransaction.test.ts:243]: Should remove the `.only` from here.

**Review by shekohex — COMMENTED** [review]
LGTM! 👍

File: tests/test/evm/vanchorPrivateTransaction.test.ts [hunk 2]
```diff
@@ -671,6 +671,6 @@ describe('Vanchor Private Tx relaying with mocked governor', function () {
   after(async () => {
     await localChain1?.stop();
     await localChain2?.stop();
-    await webbRelayer?.stop();
+    //await webbRelayer?.stop();
```
Comment (salman01zp) [inline tests/test/evm/vanchorPrivateTransaction.test.ts:674]: undo this comment

Comment (salman01zp) [inline tests/test/evm/vanchorPrivateTransaction.test.ts:674]:
```suggestion
    await webbRelayer?.stop();
```

File: tests/test/evm/vanchorPrivateTransaction.test.ts (hunk 1 above)
Comment (Nutomic) [reply tests/test/evm/vanchorPrivateTransaction.test.ts]: Fixed.

File: tests/test/evm/vanchorPrivateTransaction.test.ts (hunk 2 above)
Comment (Nutomic) [reply tests/test/evm/vanchorPrivateTransaction.test.ts]: Fixed.

**Review by salman01zp — APPROVED** [review]
LGTM..!

**Pull request merged**